src/router.rs

```diff
@@ -96,9 +96,16 @@ impl Router {
         self.route(method::Options, glob, handler)
     }
 
+    /// Route will match any method, including gibberish.
```

Comment:
Sorry for the two-phase review here - it would be good to note here in the docs that handlers added on the same route for specific methods will be preferred to handlers registered with any.